matrix/count_islands_in_matrix.py

```diff
@@ -4,20 +4,21 @@
 
 
 class matrix:  # Public class to implement a graph
-    def __init__(self, row: int, col: int, graph: list):
+    def __init__(self, row: int, col: int, graph: list[list[bool]]) -> None:
         self.ROW = row
         self.COL = col
         self.graph = graph
 
-    def is_safe(self, i, j, visited) -> bool:
+    def is_safe(self, i: int, j: int, visited: list[list[bool]]) -> bool:
         return (
             0 <= i < self.ROW
             and 0 <= j < self.COL
             and not visited[i][j]
             and self.graph[i][j]
         )
 
-    def diffs(self, i, j, visited):  # Checking all 8 elements surrounding nth element
+    def diffs(self, i: int, j: int, visited: list[list[bool]]) -> None:
```

Comment:
What is the return type of this function? It can't be `None`

Reply:
There is no return statement for that function.

Reply:
Oh, apologies I combined two functions by accident